Kernel/Graphics/VirtIOGPU/Console.cpp

```diff
@@ -65,10 +65,13 @@ void Console::enqueue_refresh_timer()
                 .width = (u32)rect.width(),
                 .height = (u32)rect.height(),
             };
-            g_io_work->queue([this, dirty_rect]() {
+            auto work_item_creation_result = g_io_work->try_queue([this, dirty_rect]() {
                 m_framebuffer_device->flush_dirty_window(dirty_rect, m_framebuffer_device->current_buffer());
                 m_dirty_rect.clear();
             });
+            if (work_item_creation_result.is_error()) {
```

Comment:
Seems like we should assert in these cases if we don't know how to meaningfully handle these errors at this time.

Reply:
Right, so what do you suggest to do? keep the if statement and assert there or use MUST() wrapper?

Reply:
I think `MUST(..)` would be fine.